pkgs/by-name/ro/rockcraft/package.nix

```diff
@@ -26,20 +63,21 @@ python3Packages.buildPythonApplication rec {
       --replace-fail "distutils.util" "setuptools.dist"
```

Comment:
How come this doesn't lead to the same error?

Reply:
Seems like it's still used in `rockcraft` - at least for now: https://github.com/canonical/rockcraft/blob/8042e14b27120278cdf4ffd1d66447b439588857/rockcraft/utils.py#L25

Reply:
Yes but I'm just wondering what brings in setuptools as a dependency because it's not in rockcraft's `dependencies`.

Reply:
Figured it out: it's from launchpadlib.